app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<%= link_to("log out", destroy_admin_session_path, method: :delete) %>
+<nav class="navbar navbar-dark navbar-expand-lg bg-dark flex-md-nowrap p-0 shadow">
```

Comment:
WDYT about moving this to a partial view and render it form the `application layout`?

This way we'd have the navbar in all view that uses this layout and in case we create another one, it'd be easy to use it too.